components/Org/OrgTabs.js

```diff
@@ -1,73 +1,51 @@
 import { Tabs } from 'antd'
 import PropTypes from 'prop-types'
 import React from 'react'
-import { FormattedMessage } from 'react-intl'
 import MemberSection from '../Member/MemberSection'
 import { ProfilePanel } from '../VTheme/Profile'
 import { OrgAboutPanel } from './OrgAboutPanel'
 import Html from '../VTheme/Html'
 import VTabs from '../VTheme/VTabs'
-const { TabPane } = Tabs
-
-const orgTab =
-  <FormattedMessage
-    id='orgTabs.about'
-    defaultMessage='About'
-    description='Tab label on OrgDetailsPage'
-  />
-
-const orgMemberTab =
-  <FormattedMessage
-    id='orgMembers'
-    defaultMessage='Members'
-    description='show opportunities list on volunteer home page'
-  />
-
-const orgInstructionTab =
-  <FormattedMessage
-    id='orgInstructions'
-    defaultMessage='Getting Started'
-    description='show opportunities list on volunteer home page'
-  />
+import { OrgHistoryPanel } from './OrgHistoryPanel'
+import { orgTab, orgMemberTab, orgInstructionTab, orgOffersTab, orgEditTab, orgHistoryTab } from './OrgTabs.messages'
 
-const orgOffersTab =
-  <FormattedMessage
-    id='orgOffers'
-    defaultMessage='Offers'
-    description='show opportunities list on volunteer home page'
-  />
-
-const orgEditTab =
-  <FormattedMessage
-    id='orgTabs.edit'
-    defaultMessage='Edit'
-    description='Tab label for org Editor panel on organisation page'
-  />
+const { TabPane } = Tabs
 
 // Warning do not try to group tabs under an isFlag TabPanes must be direct Children of Tabs.
-export const OrgTabs = ({ org, onChange, canManage, defaultTab, isAuthenticated }) => (
+export const OrgTabs = ({
+  org,
+  onChange,
+  canManage,
+  defaultTab,
+  isAuthenticated
+}) => (
   <VTabs defaultActiveKey={defaultTab} onChange={onChange}>
-    <TabPane tab={orgTab} key='about'>
+    <TabPane tab={orgTab} key='about' orgTab='about'>
```

Comment:
what is the purpose of the extra orgTab property?

Reply:
I needed to be able to get a specific `<TabPane />` in unit tests - I first tried using `wrapper.find('TabPane[key="history"]')` but it turns out you can't use `key` in attribute selectors. This is covered briefly in the Enzyme documentation, see the "React Key and Ref Props" section on this page: https://enzymejs.github.io/enzyme/docs/api/selector.html specifically:

> While in most cases, any React prop can be used, there are exceptions. The key and ref props will never work; React uses these props internally.

So I added the `orgTab` attribute so I could get the history `<TabPane />`. I added the `orgTab` attribute to the other tab panes as well for consistencies sake.